website/cue/reference/components/sinks/humio.cue

```diff
@@ -127,6 +127,30 @@ components: sinks: _humio: {
 				syntax: "literal"
 			}
 		}
+		index: {
+			common:      false
+			description: "The name of the index where to send the events to. If not specified, the default index is used."
+			required:    false
+			warnings: []
+			type: string: {
+				default: null
+				examples: ["{{ host }}", "custom_index"]
+				syntax: "template"
+			}
+		}
+		indexed_fields: {
+			common:      true
+			description: "Fields to be added to the index via [Splunk HEC indexed fields](\(urls.splunk_hec_indexed_fields))."
```

Comment:
I would have this more closely match Humio's description of how they use this field:

> JSON object containing extra fields to the event. This can be used if event is a string and it is pre-processed prior to ingest to extract fields. Tags #tags can be added to the event by specifying fields starting with #.

See https://library.humio.com/preview/docs/ingesting-data/log-shippers/hec/index.html

Reply:
True, good find and totally agree!